tests/failpoints/cases/test_kv_service.rs

```diff
@@ -36,3 +37,92 @@ fn test_kv_scan_memory_lock() {
     assert!(!resp.get_error().get_abort().is_empty());
     fail::remove("raftkv_async_snapshot_err");
 }
+
+#[test]
+fn test_scan_lock_push_async_commit() {
```

Comment:
How about adding a case that scan requests reach first and async prewrite is executed with smaller `startTS`, checking the returned `min_commit_ts`.

Reply:
In this case I did two scan lock requests, one is before the prewrite request, and another meets the memory lock of the prewrite request. I think the second one is equivalent to the case you said. How do you think?

Reply:
I see, ignore this comment~